pkg/k8s/apis/cilium.io/client/crds/v1alpha1/cilium.io_tracingpoliciesnamespaced.yaml

```diff
@@ -576,6 +576,9 @@ spec:
                       - generated_syscalls
                       - generated_ftrace
                       type: string
+                    validated:
+                      description: List was validated
+                      type: boolean
```

Comment:
So from what I can tell, this option means that no validation will be performed.
Maybe `ignore-validation` or `force-no-validation` is a better name?

Reply:
it's there to ensure the validation goes through just once, so it gets validated,
sets the flag so another sensor won't run it again, I'll update changelog with that, sry

Reply:
is there a way we could avoid implementation things to be visible/editable from the UI (the yaml)?

this is typically that could go into the `status` of the resource instead of the `spec`.